### DISCUSSION TITLE
ERROR illetve WARNING válaszok

### DISCUSSION THREAD

**Discussion by Edi1978** [nav-gov-hu/eVAT#12] [Ötletek - gépi interfész]
Az ERROR ill. WARNING válaszokkal kapcsolatban lenne ötletünk.

Miért nem ERROR választ ad, ha: „A vállalkozás nem számára kiállított számlára alapoz adólevonást." Ez a dokumentáció alapján Warning lesz.
 pl.: csoportos Áfa? … a NAV tudja hogy csoportos ÁFA alany vagyok-e, ezért erre adhatna ERROR-t is akár.

A validálásnak lehetne többféle válasz eredménye, A WARNING-ot hasznos lenne több alkategóriára lehetne osztani, a súlyosságától függően.

A validációk közül azokat, amelyek kizárólag a beküldött adattartalom alapján számolhatóak, jó lenne, ha a NAV publikálná, minél könnyebben algoritmizálható formában, mert azzal időt nyernénk, ha ezeket az ellenőrzéseket már az első M2M API beküldése előtt a könyvelőprogramban le tudnánk futtatni.

**Comment by NTCA-HM** [comment]
Az ERROR és WARNING válaszok kialakítása folyamatban van. A korábban közzétett anyagokban szereplőkhöz képest még biztosan lesznek változások, módosítások és új válasz üzenetek is megképzésre kerülnek.
